sdk/digitaltwins/Azure.DigitalTwins.Core/api/Azure.DigitalTwins.Core.netstandard2.0.cs

```diff
@@ -189,43 +189,26 @@ public enum AdtDataType
         AdtBool = 0,
         AdtNumber = 1,
         AdtString = 2,
-        AdtPrimative = 3,
+        AdtPrimitive = 3,
```

Comment:
I think we will get the same feedback about the usage of an abbreviation here. 
Let's call them DigitalTwinsBool, DigitalTwinsString etc.